src/e2e-test/features/bigquery/sink/BigQueryToBigQuerySink.feature

```diff
@@ -401,8 +401,8 @@ Feature: BigQuery sink - Verification of BigQuery to BigQuery successful data tr
     Then Verify the pipeline status is "Succeeded"
     Then Validate the data transferred from BigQuery to BigQuery with actual And expected file for: "bqInsertExpectedFile"
 
-  @BQ_TIME_STAMP_SOURCE_TEST @BQ_SINK_TEST @EXISTING_BQ_CONNECTION
-  Scenario: Verify successful record transfer for the Insert operation from BigQuery source plugin to BigQuery sink with partition type Time.
+  @BQ_TIME_SOURCE_TEST @BQ_SINK_TEST @EXISTING_BQ_CONNECTION
+  Scenario: Verify successful record transfer for the Insert operation from BigQuery source plugin to BigQuery sink with partition type Time and partition field is date.
```

Comment:
bqPartitionFieldTime is used in line 435 but the scenario is related to partition filed is date , Can you add a value like 'bqPartitionFieldDate' in pluginParam properties and use the same for this scenario .

Reply:
Added the field bqPartitionFieldDate